tests/Contrib/Unit/PrometheusExporterTest.php

```diff
@@ -15,7 +14,7 @@ class PrometheusExporterTest extends TestCase
 {
     public function testEmptyMetricsExportReturnsSuccess()
     {
-        $exporter = new PrometheusExporter($this->createMock(CollectorRegistry::class));
+        $exporter = new \OpenTelemetry\Contrib\Prometheus\PrometheusExporter($this->createMock(CollectorRegistry::class));
```

Comment:
is this an auto-generated change? I think it should have a "use"...

Reply:
Yap, it's probably caused by auto-refactoring. I'll fix it.